plugins/main/public/components/endpoints-summary/services/get-outdated-agents.ts

```diff
@@ -21,7 +21,7 @@ export const getOutdatedAgentsService = async ({
     params: {
       ...(agentIds?.length || q
         ? {
-            q: finalQ,
+            q: finalQ ? `(${finalQ})` : '',
```

Comment:
**nitpick**: the usage of the parenthesis should not be necessary because the query should mean the same. Is this a bug on the Wazuh server API? If this is the case, then the problem should be solved on the server API instead of forcing to the Wazuh plugin to do a query that should return the expected results.